packages/react-native-bpk-theming/src/BpkThemeProvider.js

```diff
@@ -22,6 +22,16 @@ import createWithTheme from './create-with-theme';
 
 const BpkThemeProvider = ThemeProvider;
 
+// TODO: Upgrade theming package and remove this: https://gojira.skyscanner.net/browse/BPK-3639
```

Comment:
I added this because this causes a lot of warnings making it really hard to see anything important. The same thing happens with `@skyscanner/react-native-transitiongroup` and `recyclerlistview` but those are only used in the carousel and not really a big issue. I have also added cards to upgrade those

Reply:
Should we be having a link to our internal tools here in the source?